test/caching.bats

```diff
@@ -163,6 +163,36 @@ EOF
     [[ ! "${output}" =~ ^(.*found cached layer bind-test)$ ]]
 }
 
+@test "bind as struct" {
+    cat > stacker.yaml <<"EOF"
+bind-test:
+    from:
+        type: oci
+        url: ${{CENTOS_OCI}}
+    import:
+        - tree1/foo/bar
```

Comment:
I think this can be simplified. we don't need an import of tree1 to test binding tree2. I think the approach we want is to bind in some path, then make a change (add a file, say) in the stacker `run` section, then after the stacker run completes, we check if that new file exists at the path we bound in. we only need one path, not both tree1 and tree2